data/lewisushindi.json

```diff
@@ -5,9 +5,14 @@
   "bio": "Front End Developer | Open source contributor",
   "avatar": "https://github.com/lewisushindi.png",
   "links": [
+     {
+      "name": "Follow me on Github",
+      "url": "https://github.com/lewisushindi",
+      "icon": "FaGithub"
+    },
     {
       "name": "Follow me on Twitter",
-      "url": "https://twitter.com/lewisushindi",
+      "url": "https://twitter.com/coder_flame",
```

Comment:
You have added a duplicate `key:url`  you can remove one of the URL field to validate json

Reply:
No, it's not falling due to that. It was some internal issue. I will merge it. They can chnage this in another PR.